src/casanovoutils/summarize_mgf.py

```
@@ -0,0 +1,1247 @@
+                ],
+            )
+        else:
+            lengths_html = "<p class='note'>No annotated spectra.</p>"
+        lengths_section = (
+            f"<h2>Peptide Lengths</h2>{lengths_html}{_img(lengths_png)}"
+        )
+
+    coverage_section = ""
+    if coverage_png is not None:
+        max_charge_label = (
+            "precursor charge" if max_charge == "max" else "precursor charge − 1"
```

Ruff (0.15.2)

[warning] String contains ambiguous `−` (MINUS SIGN). Did you mean `-` (HYPHEN-MINUS)?

(RUF001)